RecoTracker/MkFit/python/customizeHLTIter0ToMkFit.py

```diff
@@ -68,12 +68,11 @@ def customizeHLTIter0ToMkFit(process):
     )
 
     process.HLTDoLocalStripSequence += process.hltSiStripRecHits
-    process.HLTIterativeTrackingIteration0.replace(process.hltIter0PFlowCkfTrackCandidates,
-                                                   process.hltIter0PFlowCkfTrackCandidatesMkFitSiPixelHits +
-                                                   process.hltIter0PFlowCkfTrackCandidatesMkFitSiStripHits +
-                                                   process.hltIter0PFlowCkfTrackCandidatesMkFitEventOfHits +
-                                                   process.hltIter0PFlowCkfTrackCandidatesMkFitSeeds +
-                                                   process.hltIter0PFlowCkfTrackCandidatesMkFit +
-                                                   process.hltIter0PFlowCkfTrackCandidates)
+
+    replaceWith = process.hltIter0PFlowCkfTrackCandidatesMkFitSiPixelHits + process.hltIter0PFlowCkfTrackCandidatesMkFitSiStripHits + process.hltIter0PFlowCkfTrackCandidatesMkFitEventOfHits + process.hltIter0PFlowCkfTrackCandidatesMkFitSeeds + process.hltIter0PFlowCkfTrackCandidatesMkFit + process.hltIter0PFlowCkfTrackCandidates
```

Comment:
please add linebreaks for readability (to fit in under 100 chars)

Reply:
I tried that at first, but python does not allow line breaks for that expression

Reply:
linebreaks are allowed inside braces.
wouldn't `replaceWith = ( ` work?

Reply:
I was actually testing just that just before I headed out for lunch